server/src/main/java/com/linecorp/centraldogma/server/ZoneResolver.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.server;
+
+import java.lang.reflect.InvocationTargetException;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.common.util.SystemInfo;
+
+final class ZoneResolver {
```

Comment:
`classpath` seems useful. That said, I decided not to add `classpath` support but use a custom `ConfigValueConverter` to dynamically load a zone name.
It would take some time for me to design whether to call the function by convention such as one method with no-args or provide an API. Let me revisit it when I come to a conclusion.